README.md

```diff
@@ -14,21 +14,12 @@ Takeaway Challenge
 
  ```
 
-Instructions
+Steps
 -------
-
-* Feel free to use google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday morning
+I began by attempting to use domain modelling techniques to help me to visualise what my code would look like. I wasn't always able to stick to this, but it did help to start like this. I tried to complete everything test first and for the most part I succeeded, though sometimes I found it hard not to work on multiple steps at once. I managed to complete all the tasks in some sense, though I was unable to work out how to use doubles. Also, I could not get the tests for the twilio text messages to work, supposedly because I needed to pay for a number from which the text would be sent. Instead, I commented out this code, and had the method simply return 'hello' in order to test that my general code worked the way I wanted it to.
```

Comment:
Nice. Thanks for this. Good to have an overview, and reminder to me that I need to update my README.md in future!